IamCredentials/src/V1/Gapic/IAMCredentialsGapicClient.php

```diff
@@ -18,11 +18,11 @@
 /*
  * GENERATED CODE WARNING
  * Generated by gapic-generator-php from the file
- * https://github.com/google/googleapis/blob/master/google/iam/credentials/v1/iamcredentials.proto
+ * https://github.com/googleapis/googleapis/blob/master/google/iam/credentials/v1/iamcredentials.proto
  * Updates to the above are reflected here through a refresh process.
  */
 
-namespace Google\Cloud\Iam\Credentials\V1\Gapic;
+namespace Google\Iam\Credentials\V1\Gapic;
```

Comment:
We need to explicitly change this back to `Google\Cloud\Iam\Credentials\V1` using the `php_namespace` proto option.